src/main/python/afp_cli/client.py

```diff
@@ -44,6 +54,9 @@ def call_api(self, url_suffix):
             else:
                 raise APICallError("API call to AWS (%s) failed: %s" % (
                     url_orig, api_result.json()['message']))
+        if not os.path.exists(self.user_config_dir):
+            os.mkdir(self.user_config_dir)
+        pickle.dump(self.session.cookies, open(self.cookie_filepath, 'w'))
```

Comment:
To make the tests pass with Python3, open() the file in mode 'wb' instead of 'w'. Similarly, when reading the cookie from disk, open the file in mode 'rb'.